README.md

```diff
@@ -26,77 +26,119 @@ The Pulsar Node.js client can be used to create Pulsar producers and consumers i
 This library works only in Node.js 10.x or later because it uses the
 [node-addon-api](https://github.com/nodejs/node-addon-api) module to wrap the C++ library.
 
-## How to install
+## Getting Started
 
-> **Note**
->
-> These instructions are only available for versions after 1.8.0. For versions previous to 1.8.0, you need to install the C++ client first. Please switch to the corresponding version branch of this repo to read the specific instructions.
-
-### Use `npm`
+To use the Pulsar Node.js client in your project, run:
 
 ```shell
 npm install pulsar-client
 ```
 
-### Use `yarn`
+for `npm` users or
```

Comment:
```suggestion
or
```

I believe either `npm` users or `yarn `users can read this section. And basically we write:

```
npm:

... npm stuff

yarn:

... yarn stuff
```

instead of here:

```
... npm stuff

for npm users

... yarn stuff

for yarn users
```